x-pack/plugins/apm/public/components/shared/links/apm/service_link/service_max_groups_message.tsx

```diff
@@ -17,7 +17,7 @@ export function ServiceMaxGroupsMessage({
 }) {
   return (
     <FormattedMessage
-      defaultMessage="Number of services that have been instrumented has reached the current max. capacity that can be handled by the APM server. There are at least {serviceOverflowCount, plural, one {1 service} other {# services}} missing in this list. Please increase the memory allocated to APM server."
+      defaultMessage="The APM server has reached the current maximum number of instrumented service metric groups that it can handle. There {serviceOverflowCount, plural, one {is} other {are}} at least {serviceOverflowCount, plural, one {1 service metric group} other {# service metric groups}} bucketed under a common bucket at the top of this list."
```

Comment:
"bucketed under a common bucket" also seems off to me

Reply:
Regarding 1st point, it would also simplify the sentence construction as well. So yes, the 1st sentence could be `The APM Server has reached its maximum capacity`.
Regarding second point, open for copy suggestions. 
@mdbirnstiehl May be we can simplify and reword the 2nd part of the sentence.

Reply:
can we just say `grouped in a common bucket at the top of this list`?